Index: layout.css
===================================================================
RCS file: /web/www/www/layout.css,v
retrieving revision 1.156
diff -U 2 -r1.156 layout.css
--- layout.css	22 Mar 2013 03:25:55 -0000	1.156
+++ layout.css	22 Mar 2013 16:34:05 -0000
@@ -121,5 +121,11 @@
 #navigation li a:hover{ background-color: maroon; color: yellow;  }
 
-#joinfsftab a{display: inline-block; font-weight: bold; font-size: 1.2em }
+#joinfsftab a{
+  text-decoration: none; color: #333;
+  padding: 0.25em; display: inline-block;
+  font-weight: bold; font-size: 1.4em;
+}
+#joinfsftab a:hover { color: red; }
+p#joinfsftab { background-color: #aacb50; }
 
 #content h2 {
Index: server/body-include-2.html
===================================================================
RCS file: /web/www/www/server/body-include-2.html,v
retrieving revision 1.46
diff -U 2 -r1.46 body-include-2.html
--- server/body-include-2.html	22 Mar 2013 03:45:50 -0000	1.46
+++ server/body-include-2.html	22 Mar 2013 16:34:05 -0000
@@ -9,5 +9,7 @@
 
 	<div id="fssbox">
-	  <p id="joinfsftab"><a href="https://www.fsf.org/associate/support_freedom?referrer=4052">Join&nbsp;the&nbsp;FSF!</a></p>
+<p class="button" id="joinfsftab">
+<a href="https://www.fsf.org/associate/support_freedom?referrer=4052">Join&nbsp;the&nbsp;FSF!</a>
+</p>
 	  <h4>Sign up for the <a href="http://www.fsf.org/fss">Free Software Supporter</a></h4>
 	  <p>A monthly email newsletter about GNU and Free Software</p>
